downloadDirectory/src/main/java/org/reactome/release/downloadDirectory/GenerateGOAnnotationFile/GOAGeneratorUtilities.java

```diff
@@ -77,26 +75,26 @@ public static boolean validateCatalystPE(GKInstance catalystPEInst) throws Excep
     /**
      * Checks if the PhysicalEntity is a 'multi-instance' type, which includes Complexes, EntitySets, and Polymers
      * @param physicalEntitySchemaClass
-     * @return
+     * @return == true/false indicating if this instance has multiple subunits that comprise it.
```

Comment:
It might be good to rename this method "isMultiInstancePhysicalEntity".  Convention for boolean methods is to have them start with either the word "is" or "has", depending on which is appropriate, whenever possible.

Reply:
Done